src/Symfony/Component/Templating/PhpEngine.php

```diff
@@ -330,6 +330,9 @@ public function escape($value, $context = 'html')
      */
     public function setCharset($charset)
     {
+        if ('UTF8' === $charset = strtoupper($charset)) {
```

Comment:
it's because iconv on windows doesn't accept utf8 as an encoding name: only utf-8 is supported